actionpack/lib/action_dispatch/system_testing/driver.rb

```diff
@@ -37,6 +38,16 @@ def register_selenium(app)
           end
         end
 
+        def register_selenium_headless_chrome(app)
+          browser_options = Selenium::WebDriver::Chrome::Options.new
+          browser_options.args << "--headless"
+          browser_options.args << "--disable-gpu"
+
+          Capybara::Selenium::Driver.new(app, { browser: :chrome, options: browser_options }.merge(@options)).tap do |driver|
+            driver.browser.manage.window.size = Selenium::WebDriver::Dimension.new(*@screen_size)
+          end
+        end
```

Comment:
Let's follow the pattern of other drivers instead. Selenium is still the default driver, and then the option for the browser would be `:headless_chrome`. Which means instead of `driven_by :selenium_headless_chrome` this should be `driven_by :selenium, using: :headless_chrome`.

Reply:
Thanks for your review! It makes sense. I updated to use `using` option.